examples/speech_to_text/eval_whipser_waitk.sh

```diff
@@ -0,0 +1,6 @@
+simuleval \
+    --agent whisper_waitk.py \
+    --source-segment-size 500 \
+    --waitk-lagging 3 \
+    --source source.txt --target reference/transcript.txt \
+    --output output --quality-metrics WER
```

Comment:
Do we need the eval bash script if it's just one command that's already in the README?